frontend/components/events/create/forms/BasicForm.tsx

```diff
@@ -61,41 +70,59 @@ export function BasicInformation({
       updateField({ isPrivate: true });
     }
   };
-  const handleDateChange = (selectedDate: string) => {
-    // Validate if the selected date is in the past
-    const currentDate = new Date();
-    const selectedDateObj = new Date(selectedDate);
-
-    if (selectedDateObj < currentDate) {
-      setDateWarning("Selected date is in the past.");
-    } else {
-      setDateWarning(null);
-    }
+  const handleStartDateChange = (selectedDate: string) => {
+    updateField({ startDate: selectedDate });
+  };
 
-    updateField({ date: selectedDate });
+  const handleEndDateChange = (selectedDate: string) => {
+    updateField({ endDate: selectedDate });
   };
 
-  const handleStartTimeChange = (selectedTime: string) => {
-    // Validate if end time is before start time
-    if (endTime && selectedTime >= endTime) {
-      setTimeWarning("End time must be after start time.");
-    } else {
-      setTimeWarning(null);
-    }
+  useEffect(() => {
```

Comment:
whats the point of this?

Reply:
end date automatically gets changed to start date when start date is changed - this is to help avoid errors with incorrect architecture of dates - most events are one day as well